Demo/Common/IntegrationExamples/Components/CardComponentAdvancedFlowExample.swift

```diff
@@ -33,34 +33,35 @@ internal final class CardComponentAdvancedFlowExample: InitialDataAdvancedFlowPr
 
     internal init() {}
 
-    // MARK: - Networking
-
-    internal func requestInitialData(completion: ((PaymentMethods?, Error?) -> Void)?) {
-        requestPaymentMethods(order: nil) { [weak self] paymentMethods, errorResponse in
-            guard paymentMethods != nil else {
-                guard let errorResponse = errorResponse else {
-                    return
+    internal func present() {
```

Comment:
Let's split network and present logics into different functions, i.e., here the completion block after hiding the spinner can be its own function.

So like a "load" function then "present"

Reply:
d6e580c29b4b8c6d9048e53bfc295488ddf025a6